mobile/lib/features/trade/domain/order.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:get_10101/features/trade/domain/contract_symbol.dart';
+import 'package:get_10101/features/trade/domain/leverage.dart';
+
+import 'package:get_10101/features/trade/domain/direction.dart';
+import '../../../bridge_generated/bridge_definitions.dart' as rust;
```

Comment:
🤔 It looks inconsistent that we use the `rust` alias here as mostly we use the rust alias for the `ffi.dart` package. How about `bridge` as alias here?

Also can you change the path to a package import?